blog/2024-01-llms-graphs-part-2/index.md

```diff
@@ -292,6 +292,8 @@ to extract high quality triples. Or you can just default to thinking hard about
 so what type of questions you want to answer in your RAG system, and based on those give example documents
 and triples and train a specialized model. 
 
+[^4]: SpanMarker is a recent approach for training powerful Named Entity Recognition models and is tightly implemented on top of the Transformers library by Hugging Face. It is based on the [PL-Marker paper](https://arxiv.org/pdf/2109.06067.pdf). The idea is to train a model that can detect NER entities from a large corpus of documents., label them with their types and map these types to the required triples as per your business domain.
+
 
 ## Agents: Developing RAG Systems That Use Both Structured & Unstructured Data
 Let me now start wrapping up. In my last post and this one, I covered RAG systems using structured and unstructured data.
```

Comment:
In this section, the statement isn't very clear:

> "if the question is about foo retrieve fact from the GDBMS by converting $Q_{NL}$ to a Cypher query; otherwise follow the RAG-U pipeline to retrieve chunks from documents."

What is the meaning of "foo" here? Am I missing something?

Reply:
I was just using it as a placeholder as in: if the question is about x do such and such, otherwise if it's about y, do such and such. I'll change it.